hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/JdbcBackendResolver.java

```diff
@@ -79,7 +86,17 @@
     }
     entityBuilder.putAttributes(
         RawSpanConstants.getValue(Sql.SQL_DB_TYPE), createAttributeValue(dbType));
-    return Optional.of(entityBuilder.build());
+
+    setAttributeForFirstExistingKey(
```

Comment:
if the attribute was not added to the entity before, let's not add it to the entity